### DISCUSSION TITLE
00A49425-F9E4-9E59-B578-27DDC0B3FE3D

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#46] [General]
# 00A49425-F9E4-9E59-B578-27DDC0B3FE3D

This addon adds a catalog containing 1,303 asteroids discovered in 2022. 1,473 are NEOs and another 2 are main belt asteroids.

https://celestia.mobi/addon?item=00A49425-F9E4-9E59-B578-27DDC0B3FE3D&lang=zh_CN

**Comment by 13940951572** [comment]
出一款2023年近地小行星

**Comment by 13940951572** [comment]
本月24号有2023 CL3小行星靠近地球！

**Comment by Usvgamer222yt** [comment]
This one I love it really helps me crash the game :D
